jest.config.js

```diff
@@ -74,8 +74,8 @@ module.exports = {
     "\\.(css|less|scss)$": "identity-obj-proxy",
     "^\\$web-common\\/(.*)": "<rootDir>/components/common/$1",
     "^\\$web-components\\/(.*)": "<rootDir>/components/web-components/$1",
-    "^brave-ui$": "<rootDir>/node_modules/brave-ui/src",
-    "^brave-ui\\/(.*)": "<rootDir>/node_modules/brave-ui/src/$1",
+    "^brave-ui$": "<rootDir>/node_modules/@brave/brave-ui",
+    "^brave-ui\\/(.*)": "<rootDir>/node_modules/@brave/brave-ui/$1",
```

Comment:
Don't need to do it here, but I wonder if we should get rid of these import maps. I don't really think they add anything :laughing:

Reply:
This one is needed to map brave-ui to @brave/brave-ui, or do you mean something else?

Reply:
Yeah, exactly - it just seems like an unnecessary bit of indirection. But not that important, and I don't think we need to do anything here.

Reply:
Without it, we'd need to find/replace all uses of 'brave-ui/xyz' with '@brave/brave-ui/xyz' - it's similar to the ones we have for webpack and typescript.